test/common/http/http1/codec_impl_test.cc

```diff
@@ -310,8 +321,68 @@ TEST_F(Http1ServerConnectionImplTest, ChunkedBody) {
   EXPECT_CALL(decoder, decodeData(BufferEqual(&expected_data), false)).Times(1);
   EXPECT_CALL(decoder, decodeData(_, true)).Times(1);
 
-  Buffer::OwnedImpl buffer(
-      "POST / HTTP/1.1\r\ntransfer-encoding: chunked\r\n\r\nb\r\nHello World\r\n0\r\n\r\n");
+  Buffer::OwnedImpl buffer("POST / HTTP/1.1\r\ntransfer-encoding: chunked\r\n\r\n"
+                           "6\r\nHello \r\n"
+                           "5\r\nWorld\r\n"
+                           "0\r\n\r\n");
+  codec_->dispatch(buffer);
+  EXPECT_EQ(0U, buffer.length());
+}
+
+TEST_F(Http1ServerConnectionImplTest, ChunkedBodySplit) {
```

Comment:
Nit: please add a short `//` above each new `TEST_?` method explaining the intuition of what is being tested.

Reply:
Done.